### PR TITLE
[sqllab] making 'click to retrieve results' a button

### PR DESCRIPTION
**Pull request by mistercrunch**

### CODE REVIEW

File: superset/assets/javascripts/SqlLab/components/ResultSet.jsx [hunk 1]
```diff
@@ -211,12 +211,13 @@ class ResultSet extends React.PureComponent {
     }
     if (query.cached) {
       return (
-        <a
-          href="#"
+        <Button
```
Comment (bkyryliuk) [inline superset/assets/javascripts/SqlLab/components/ResultSet.jsx]: I think there was a bug in this functionality that query wasn't using the cta schema, could u please fix it in this PR ?

Comment (mistercrunch) [inline superset/assets/javascripts/SqlLab/components/ResultSet.jsx]: is there an issue documenting the bug?

Comment (bkyryliuk) [inline superset/assets/javascripts/SqlLab/components/ResultSet.jsx]: I think it's related to the https://github.com/airbnb/superset/issues/1708

**Comment by mistercrunch** [comment]
This is just a simple cosmetic change, let's cut another PR for the issue you are referring to.

**Pull request merged**